SPECS-EXTENDED/azl-otel-collector/azl-otel-collector.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Azure Linux OTEL Collector
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/azl-otelcol --config /etc/azl-otel-collector/config.yaml
+KillMode=process
+Restart=on-failure
+RestartSec=60s
```

Comment:
would it make sense to make the restart after a shorter period of time (1 sec) or is 60s intentional?

Reply:
hm I just put 60s as general time, but yes I think a shorter period is better, I'll make it 5sec